assimilation_code/modules/assimilation/filter_mod.f90

```diff
@@ -2719,6 +2730,69 @@ subroutine set_copies(cnum, STAGE_COPIES)
 
 end subroutine set_copies
 
+
+!-------------------------------------------------------------------------------
+!> The infl_flavor namelist is a string, and specifies the inflation algorithm.
+!> The character string can either be the name associated with the type of inflation
+!> or the integer associated with the type of inflation. The string names of the
+!> inflation algorithms is based on what is declared in the adaptive_inflate_mod.f90
+!> which is repeated here for reference. 
+!>
+!> NO_INFLATION               = 0
+!> OBS_INFLATION              = 1    observation-space inflation (deprecated)
+!> VARYING_SS_INFLATION       = 2    spatially-varying state-space inflation
+!> SINGLE_SS_INFLATION        = 3    spatially-constant state-space inflation
+!> RELAXATION_TO_PRIOR_SPREAD = 4    (available only with posterior inflation)
+!> ENHANCED_SS_INFLATION      = 5    Inverse Gamma version of VARYING_SS_INFLATION
+
+subroutine parse_inflation_string(flavor_string,flavor_integer)
+
+character(len=*), intent(in)  :: flavor_string(2)
+integer,          intent(out) :: flavor_integer(2)
+
+integer :: i
+character(len=32) :: string
+
+do i = 1,2   ! one is prior, two is posterior
+
+   string = adjustl(flavor_string(i))
+   call to_upper(string)
+
+   select case(trim(string))
+      case ('0','NO_INFLATION')
```

Comment:
Should we add more options? For instance: 
- case('0', 'NO_INFLATION', 'no inf') 
-case('1', 'OBS_INFLATION', 'obs inf') 
- case('2', 'VARYING_SS_INFLATION', 'A09') 
- case('5', 'ENHANCED_SS_INFLATION', 'E18')?
I understand this might complicate the documentation a bit. I just like to write a short version of the word "inflation" as "inf". A09 and E18 refer to Anderson 2009 and El Gharamti 2018, respectively.

Reply:
This is discussion for the group ... at the standup today.